src/hotspot/share/runtime/thread.cpp

```diff
@@ -554,20 +554,11 @@ bool Thread::set_as_starting_thread(JavaThread* jt) {
   return os::create_main_thread(jt);
 }
 
-// Ad-hoc mutual exclusion primitives: SpinLock
+// Ad-hoc mutual exclusion primitives: spin lock
```

Comment:
```suggestion
// Ad-hoc mutual exclusion primitive: spin lock
```
We dropped the Mux primitive a long time ago, so there is only one